Automattic/pocket-casts-android
# Fix Video Controls Overlay Getting Stuck on API <30

# Description

Fixes a bug where on phones running api <30 the video player controls would be stuck on the screen

Fixes #60

# Test Steps

1. On a device/emulator running API <30 (I've only tested this using a Pixel emulator, might be good to test on a non-Google physical device if someone has one running an API < 30).
2. Start watching a video (i.e., Ted Talks has some video streams)
3. From the full screen player, tap on the video so the video appears full screen
4. Wait ~3 seconds for the on-screen media controls to be hidden
5. Tap on the screen to make the on-screen media controls re-appear
6. Wait another ~3 seconds for the on-screen media controls to be hidden again

# Checklist
